app/controllers/App.scala

```diff
@@ -1,14 +1,27 @@
 package controllers
 
+import cats.syntax.either._
 import config.Config
+import database.MetricsDB
 import play.api.Logger
 import play.api.libs.ws.WSClient
 import play.api.mvc._
+import util.Utils._
+import util.Parser._
 
-class App(val wsClient: WSClient) extends Controller with PanDomainAuthActions {
+class App(val wsClient: WSClient, val config: Config, val db: MetricsDB) extends Controller with PanDomainAuthActions {
 
   def index = AuthAction {
-    Logger.info(s"I am the ${Config.appName}")
+    Logger.info(s"I am the ${config.appName}")
     Ok(views.html.index())
   }
+
+  def createdInCopyContent = AuthAction { req =>
+    val x = for {
+      body <- extractRequestBody(req.body.asJson.map(_.toString))
+      json <- stringToJson(body)
+      data <- jsonToInCopyData(json)
+    } yield data
```

Comment:
Sorry this is a slightly strange bit of code I was experimenting with parsing so the function name is not very helpful. I would expect this to put the data into the database but I didn't ever connect up the dots in this PR as I wanted to get feedback on the quill work. It probably didn't need to go in this PR